test/addons/layout-change/lib/test-cuddlefish-loader.js

```diff
@@ -21,11 +21,6 @@ exports["test compatibility"] = function(assert) {
   assert.equal(require("tabs"),
                require("sdk/tabs"), "sdk/tabs -> tabs");
 
-  if (app.is("Firefox")) {
-    assert.equal(require("widget"),
-                 require("sdk/widget"), "sdk/widget -> widget");
-  }
```

Comment:
No, that's clear. I forgot to remove "widget" from mapping, and I have to do that in my next commit. I was questioning the first part, that we should test that the mapping does not exist. I don't agree with that, because doing so we actually introduce a code that is referring to a module that doesn't exist anymore, an soon or later we have to remove again – and we probably forget to do so.
So my point was just to remove the reference in all our codes – including mapping and unit test, without adding a unit test check for a module that doesn't exist.